public/index.html

```diff
@@ -224,6 +224,44 @@ <h2>Review</h2>
   <div class="vue-component">
     <div class="container">
       <h1>Pending Reviews</h1>
+        <h2>Manager</h2>
+
+        <select v-model="selectedManager">
+          <option value="">Please Select</option>
+          <option
+            v-for="employee in employees"
+            :value="employee.id"
+            v-if="employee.manager_status == true"
```

Comment:
API that is removed - manager index is taking care of this logic
It can be cleaned up later